centreon/www/front_src/src/ApiTokens/api/endpoints.ts

```diff
@@ -38,3 +42,9 @@ export const deleteTokenEndpoint = ({
   userId
 }: DeleteTokenEndpoint): string =>
   `${baseTokenEndpoint}/${tokenName}/users/${userId}`;
+
+export const patchTokenEndpoint = ({
```

Comment:
the patchTokenEndpoin its the same af deleteTokenEndpoint , maybe you van keep just one and give a generic name ,and then export as

Reply:
Done